src/lib/components/navbar/Navbar.component.js

```diff
@@ -48,6 +56,43 @@ const NavbarMenu = styled.div`
   align-items: center;
 `;
 
+const NavbarTabs = styled.div`
+flex: 1;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+`;
+
+const TabItems = styled.div`
+box-sizing: border-box;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;  
+  padding: ${defaultTheme.padding.base};
+  :hover{
+    border-bottom: 1vh solid black;
+    cursor: pointer;
+  }
+  ${props => 
+    css`
+        {
+          color: mergeTheme(props.theme, defaultTheme).primary
+        }
+    `}
+  ${props => {
+    return (
+      
+      props.selected &&
+      css`
+        {
+          border-bottom: 1vh solid black;
```

Comment:
it should be `mergeTheme(props.theme, defaultTheme).primary` instead of `black` and `2px ` instead of  `1vh`